unit_tests/test_Inspector.py

```diff
@@ -80,10 +88,18 @@ def test_inspector(self):
         self.assertEqual(repr(Diskfile.Diskfile(goodfile, self.dbu)), repr(self.inspect.Inspector(goodfile, self.dbu, 1,)()))
         #self.assertEqual(None, self.inspect.Inspector(goodfile, self.dbu, 1,).extract_YYYYMMDD())
         # This inspector sets the data_level - not allowed
-        inspect = imp.load_source('inspect', os.path.join(
-            dbp_testing.testsdir, 'inspector', 'rot13_L1_dlevel.py'))
-        with warnings.catch_warnings(record=True) as w:
-            self.assertEqual(repr(Diskfile.Diskfile(goodfile, self.dbu)), repr(self.inspect.Inspector(goodfile, self.dbu, 1,)()))
+        if sys.version_info < (3, 4):
+            import imp # Depracated in Python 3.4
+            inspect = imp.load_source('inspect', os.path.join(dbp_testing.testsdir, 'inspector', 'rot13_L1_dlevel.py'))
+        else:
+            import importlib.util
+            import importlib.machinery
+            loader = importlib.machinery.SourceFileLoader('inspect', os.path.join(dbp_testing.testsdir, 'inspector', 'rot13_L1_dlevel.py'))
+            spec = importlib.util.spec_from_file_location('inspect', os.path.join(dbp_testing.testsdir, 'inspector', 'rot13_L1_dlevel.py'), loader=loader)
+            inspect = importlib.util.module_from_spec(spec)
+            loader.exec_module(inspect)
+        with warnings.catch_warnings(record=True) as w: 
+            self.assertEqual(repr(Diskfile.Diskfile(goodfile, self.dbu)), repr(inspect.Inspector(goodfile, self.dbu, 1,)()))
```

Comment:
Can we break this out into a function, or static method, that gets called both here and the setup? Or I wonder about putting the function in dbprocessing/Utils.py, since this is also used by the code in dbprocessing/dbprocessing.py.